build_helper.sh

```diff
@@ -131,15 +146,32 @@ function setup_folly() {
       exit 1
     fi
   fi
+
+  if [ "$Platform" = "Mac" ]; then
+    # Homebrew installs OpenSSL in a non-default location on  MacOS >= Mojave
+    # 10.14 (because MacOS has its own SSL implementation).  If we find
+    # Homebrew's OpenSSL pkgconfig file, set it so that cmake will find/use it.
+    dir=/usr/local/opt/openssl/lib/pkgconfig
+    if [ -d $dir ]; then
+        # We want to prefix PKG_CONFIG_PATH if it exists, but with "set -u",
+        # there's no good way to check to see if $PKG_CONFIG_PATH is set or not.
+        # So temporarily turn it off.
+        set +u
+        export PKG_CONFIG_PATH="$dir:$PKG_CONFIG_PATH"
+        set -u
+    fi
+  fi
+
   echo -e "${COLOR_GREEN}Building Folly ${COLOR_OFF}"
   mkdir -p "$FOLLY_BUILD_DIR"
   cd "$FOLLY_BUILD_DIR" || exit
   cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo         \
     -DCMAKE_PREFIX_PATH="$FOLLY_INSTALL_DIR"      \
     -DCMAKE_INSTALL_PREFIX="$FOLLY_INSTALL_DIR"   \
     ..
-  make -j "$(nproc)"
+  make -j $nproc
```

Comment:
```suggestion
  make -j "$nproc"
```

Reply:
Just curious: why the quotes?  `nproc` is always going to be a single-token integer.  There's no need to "cast" it to be a string (in the shell sense).  And if `nproc` is (erroneously) more than a single token, `make` will have a difficult time parsing it, regardless of whether you force it into a single `argv` entry or not.  Is there a different case to worry about here?

Reply:
In this specific case, yes. 
We use an static code analyzer internally to check for these cases to prevent globbing and word splitting (and for better or worse it complains here) :)

Reply:
Gotcha.